### PR TITLE
chore: adds @enj to CODEOWNERS

### PR DESCRIPTION
**Pull request by nilekhc**
**Reason for Change**:

**Issue Fixed**:

**Notes for Reviewers**:

### CODE REVIEW

**Review by sozercan — APPROVED** [review]

**Review by aramase — COMMENTED** [review]
LGTM

**Review by aramase — COMMENTED** [review]
Let's wait for @enj `lgtm` before merging this!

**Pull request merged**
